build-tools/automation/azure-pipelines.yaml

```diff
@@ -293,10 +293,11 @@ stages:
       displayName: make create-nupkgs
 
     - script: >
+        df -h &&
         mkdir -p $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/nuget-linux &&
-        cp $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)/Microsoft.Android.Sdk.Linux*.nupkg
+        ln $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)/Microsoft.Android.Sdk.Linux*.nupkg
         $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/nuget-linux &&
-        cp $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)/SignList.xml
+        ln $(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/$(NuGetArtifactName)/SignList.xml
```

Comment:
I don't know if the [PublishPipelineArtifact task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact?view=azure-devops) will handle links. Unfortunately, it doesn't handle wildcards in the upload path either. We could try to use `mv` if `ln` doesn't work since there are no longer tests running as part of this build job.

Reply:
How do we know/test/verify that it works or failed?  The **copy linux sdks** step succeeded: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5726101&view=logs&j=a1070d6b-7539-53cb-46be-b3395553b4a7&t=f2a79c73-51ad-504e-a57f-523f1d9fb808

```
Script contents:
df -h && mkdir -p /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-linux && ln /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-unsigned/Microsoft.Android.Sdk.Linux*.nupkg /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-linux && ln /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-unsigned/SignList.xml /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-linux
========================== Starting Command Output ===========================
/usr/bin/bash --noprofile --norc /mnt/vss/_work/_temp/c582dc26-b907-439e-bd14-e66e32a6c897.sh
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        84G   84G  106M 100% /
devtmpfs         16G     0   16G   0% /dev
…
```

The [**upload linux sdk**](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5726101&view=logs&j=a1070d6b-7539-53cb-46be-b3395553b4a7&t=39fd3fb3-80da-5743-44e5-b4c9bf2cd58c) step is also green:

```
Artifact name input: nuget-linux-unsigned
Uploading pipeline artifact from /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-linux for build #5726101
Using default max parallelism.
Max dedup parallelism: 192
ApplicationInsightsTelemetrySender will correlate events with X-TFS-Session 756f0c54-8e20-4f13-8ecf-889599338ed7
DedupManifestArtifactClient will correlate http requests with X-TFS-Session 756f0c54-8e20-4f13-8ecf-889599338ed7
2 files processed.
Processed 2 files from /mnt/vss/_work/1/s/xamarin-android/bin/BuildRelease/nuget-linux successfully.
Uploaded 10,176,104 out of 85,651,189 bytes
Content upload is done!
```

Is this success?

Reply:
We can look at the `nuget-linux-unsigned` pipeline artifact to confirm, but seeing as it seems to have uploaded 85,651,189 bytes it looked like this worked.